t/smoke/tests/quorum_queues/test_native_delayed_delivery.py

```diff
@@ -0,0 +1,169 @@
+from datetime import timedelta
+from datetime import timezone as datetime_timezone
+
+import pytest
+import requests
+from future.backports.datetime import datetime
+from pytest_celery import CeleryTestSetup
+from requests.auth import HTTPBasicAuth
+
+from celery import Celery
+from t.smoke.tasks import noop
+from t.smoke.tests.quorum_queues.conftest import RabbitMQManagementBroker
+
+
+class test_broker_configuration_quorum:
+    @pytest.fixture
+    def default_worker_app(self, default_worker_app: Celery) -> Celery:
+        app = default_worker_app
+        app.conf.broker_transport_options = {"confirm_publish": True}
+        app.conf.task_default_queue_type = "quorum"
+        app.conf.broker_native_delayed_delivery = True
+        app.conf.broker_native_delayed_delivery_queue_type = 'quorum'
+        app.conf.task_default_exchange_type = 'topic'
+        app.conf.task_default_routing_key = 'celery'
+
+        return app
+
+    def test_native_delayed_delivery_queue_configuration(
+        self,
+        celery_setup: CeleryTestSetup,
+        default_worker_app: Celery
+    ):
+        broker: RabbitMQManagementBroker = celery_setup.broker
+        api = broker.get_management_url() + "/api/queues"
+        response = requests.get(api, auth=HTTPBasicAuth("guest", "guest"))
+        assert response.status_code == 200
+        res = response.json()
+        assert isinstance(res, list)
```

Comment:
Move this into a fixture please. It will also reduce code duplication.

Reply:
I tried moving this to a fixture but I got an error due to a bug in pytest-celery.
I think this doesn't matter much.